.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Does it matter though? It's the editor of choice for me and having it in there is rather convenient.